trillian_api.proto

```diff
@@ -65,6 +66,12 @@ message Proof {
     repeated Node proof_node = 2;
 }
 
+message QueuedLogLeaf {
```

Comment:
Documented `QueuedLogLeaf` but I'm not going to change the documentation level of the entire existing API....